lib/webdriveragent.js

```diff
@@ -348,6 +332,22 @@ class WebDriverAgent {
     if (this.mjpegServerPort) {
       xctestEnv.MJPEG_SERVER_PORT = this.mjpegServerPort;
     }
+
+    const waitForWdaStart = async () => {
```

Comment:
maybe move it to a separate method (_waitForPreinstalledServer). There is no need to increase this method length